code/game/atoms.dm

```diff
@@ -282,6 +282,11 @@ its easier to just keep the beam vertical.
 	to_chat(user, desc)
 	if(health_max)
 		examine_damage_state(user)
+	if(SCP && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/datum/job/job = SSjobs.get_by_title(H.job)
+		if(job && (job.department_flag & (COM|SCI|SEC)))
+			to_chat(user, SPAN_CLASS("scp", "You know this is SCP-[SCP.designation]!"))
```

Comment:
Maybe we can move it out somewhere else????

Reply:
might be possible to move it to the SCP component, will check

Reply:
ghosts being able to know what SCP it is would be good too